### PR TITLE
Add chainwatch systemd svc config

### PR DESCRIPTION
**Pull request by placer14**
Add chainwatch service with make install directives. Fix documentation menu links for new item.

Unsure if we want to include documentation around this. If so, I'd be happy to add it if you let me know where the best place might be.

### CODE REVIEW

**Comment by placer14** [comment]
Fixed some of the logging in the `.service` files and the link to the new systemd doc page (and verified it worked this time). Would appreciate any opinions on the service file changes.

FYI, I'm working toward managing all the sub-services of sentinel via systemd via `Requires=` and `After=` rules with the plan to do something like `systemctl start sentinel` and it will start everything needed.

**Comment by placer14** [comment]
FYI, the CI failures do not appear related to this PR.

File: scripts/chainwatch.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Chainwatch
+PartOf=sentinel.service
```
Comment (magik6k) [inline scripts/chainwatch.service:3]: I'd comment this out by default
```suggestion
# PartOf=sentinel.service
```

File: scripts/chainwatch.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+[Service]
+Environment=GOLOG_FILE="/var/log/lotus/chainwatch.log"
+Environment=GOLOG_LOG_FMT="json"
+Environment=LOTUS_DB=""
+Environment=LOTUS_PATH="%h/.lotus"
+EnvironmentFile=-/etc/lotus/chainwatch.env
+ExecStart=/usr/local/bin/chainwatch run
+
+[Install]
+WantedBy=multiuser.target
```
Comment (travisperson) [inline scripts/chainwatch.service:15]: This is a typo in the original file as well, but this should be `multi-user.target`

Comment (placer14) [reply scripts/chainwatch.service]: Good catch. Fixed. 🤝

**Pull request merged**